src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -86,11 +86,15 @@ private static MIBreakpointState StringToBreakpointState(string addr)
 
         internal static async Task<BindResult> Bind(string functionName, DebuggedProcess process, string condition, AD7PendingBreakpoint pbreak)
         {
+            process.VerifyNotDebuggingCoreDump();
```

Comment:
Since vscode uses the OpenDebug prototocol and not com, I suspect we'll need to make sure opendebugad7 catches this and displays an error. If you've already tested that an error is displayed then it is doing the right thing. The behavior you describe for breakpoints sounds reasonable.